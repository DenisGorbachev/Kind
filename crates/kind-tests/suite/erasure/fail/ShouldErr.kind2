Kek <a: Data.U60> : Data.U60
Kek a = a

Main : Data.U60
Main = (Kuk) ~(20)

Kuk : ~(a : Data.U60) -> Data.U60
Kuk = ~x => Kek 3
