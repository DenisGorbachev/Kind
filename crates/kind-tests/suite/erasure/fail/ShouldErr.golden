   ERROR  This irrelevant parameter should not be used in a relevant position.

      /--[suite/erasure/fail/ShouldErr.kind2:2:9]
      |
    1 |    Kek <a: Data.U60> : Data.U60
    2 |    Kek a = a
      |            v
      |            \It's in relevant position!
    3 |    


